modules/oe_content_publication/oe_content_publication.info.yml

```diff
@@ -5,25 +5,41 @@ package: OpenEuropa
 type: module
 core: 8.x
 dependencies:
+  - composite_reference:composite_reference
+  - datetime:datetime
```

Comment:
Datetime is already a dependency of oe_content so no need to add it here nor in the post_update.

Reply:
I removed the datetime dependency.